static/webmention.js

```diff
@@ -54,14 +54,25 @@ GitHub repo (for latest released versions, issue tracking, etc.):
         'follow-of': '🐜',
     };
 
+    var rsvpEmoji = {
+        'yes': '✅',
+        'no': '❌',
+        'interested': '💡'
+        'maybe': '⚖️'
```

Comment:
We could always use ie 🤷‍♀️ or 🤔 for maybe?

Reply:
I was considering 🤷‍♀️ but that has connotations along the lines of "I dunno" or "I'm indifferent" as opposed to "I'm considering it." 🤔 might work though, although that's taken on some other meanings too.

Reply:
haha of course, the ultimate "thinking about it" would be 💭